stylelint-polaris/tests/index.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+
+const stylelint = require('stylelint');
+
+const fixturePath = path.join(process.cwd(), 'tests/fixture.css');
```

Comment:
```suggestion
const fixturePath = path.join(__dirname, 'fixture.css');
```
`process.cwd()` will not point to the `stylelint-polaris` dir if ran from the root of the monorepo. `__dirname` should have more consistent behavior.